Doc/library/html.parser.rst

```diff
@@ -15,14 +15,17 @@
 This module defines a class :class:`HTMLParser` which serves as the basis for
 parsing text files formatted in HTML (HyperText Mark-up Language) and XHTML.
 
-.. class:: HTMLParser(*, convert_charrefs=True)
+.. class:: HTMLParser(*, convert_charrefs=True, scripting=False)
 
    Create a parser instance able to parse invalid markup.
 
    If *convert_charrefs* is ``True`` (the default), all character
    references (except the ones in ``script``/``style`` elements) are
    automatically converted to the corresponding Unicode characters.
 
+   If *scripting* is true, the ``noscript`` element is parsed in the
+   RAWTEXT mode.
```

Comment:
```suggestion
   If *scripting* is false (the default), the content of the ``noscript`` element
   is parsed normally; if it's true, it's parsed in RAWTEXT mode.
```
And this could link to the aforementioned section.

It would also be useful to expand a bit on why this exists, either by mentioning briefly that `<noscript>` is used as a fallback when JS is not enabled, or by linking to a page (w3c or mdn) that explains that.